docs/dns/README.md

```diff
@@ -0,0 +1,15 @@
+<!--- Hugo front matter used to generate the website version of this page:
+linkTitle: DNS
```

Comment:
I think it conflicts with the same title on dns-metrics doc, please update title there to be `Metrics`